components/scream/scripts/test_all_scream.py

```diff
@@ -53,6 +53,8 @@ def __init__(self, cxx, kokkos=None, submit=False, parallel=False, fast_fail=Fal
         expect(not (self._baseline_ref and self._baseline_dir),
                "Makes no sense to specify a baseline generation commit if using pre-existing baselines ")
 
+        self._custom_cmake_opts.append("EKAT_DEFAULT_BFB=ON")
```

Comment:
I would not add it to the custom cmake opts, cause those are applied to all builds. I would instead add this to all dict entries below.